src/app/containers/MediaPlayer/__snapshots__/index.test.jsx.snap

```diff
@@ -235,21 +262,25 @@ exports[`MediaPlayer Calls the canonical player when platform is canonical and p
   }
 }
 
-<div
-  class="c0"
->
-  <div
-    class="c1"
-  >
-    <div
-      class="c2"
-    >
-      <img
-        alt="Image Alt"
-        class="c3"
-        src=""
-      />
+<div>
+      <div
+        class="c0"
+      >
+        <div
+          class="c1"
+        >
+          <div
+            class="c2"
+          >
+            <img
+              alt="Image Alt"
+              class="c3"
+              src=""
+            />
+          </div>
+        </div>
+      </div>
     </div>
-  </div>
-</div>
+  </body>
+</html>
```

Comment:
Why does this snapshot now include `html` elements but it didn't before?

Reply:
@jamesdonoh I think that's because of the `Helmet` wrapper being exposed

Reply:
But which change in the implementation code would account for that? I can't see any difference in how we are using Helmet here that would account for such major changes to the structure of the snapshots.